.github/workflows/build.yml

```diff
@@ -26,15 +26,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        suffix: ['', '-headless']
-        python: ['3.12', '3.11', '3.10', '3.9']
-        node: ['20', '18', '16']
+        python: ['3.13', '3.12', '3.11', '3.10']
```

Comment:
If you want, it looks like we could remove 3.10 here too: https://github.com/search?q=org:oddbird+ghcr.io/oddbird/pyjs:py3.10&type=code

Reply:
I went ahead and removed node 18 as well, it looks like we are on 16 or 20 on all projects